src/pages/comparison/comparison.tsx

```diff
@@ -219,10 +293,14 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
         onSelectBaseMap={onSelectBaseMapHandler}
         onDeleteBaseMap={onDeleteBaseMapHandler}
         onRequestTransitionToTileset={() => setNeedTransitionToTileset(true)}
+        disableButtonHandler={disableButtonHandlerRight}
+        onTilesetLoaded={() => loadManagerRef.current.resolveRightSide()}
+        onStopTimer={() => loadManagerRef.current.stoptTimer()}
       />
       <MapControllPanel
         bearing={viewState.main.bearing}
         dragMode={dragMode}
+        compareButtonMode={compareButtonMode}
```

Comment:
Check `{compareButtonMode === CompareButtonMode.Start && (...` here instead of changing `MapControlPanel`